index.html

```diff
@@ -700,7 +751,7 @@ <h2>Verification Methods</h2>
             <h3>Verification Material</h3>
 
             <p>
-Verification material SHOULD be expressed in the <code>publicKeyJwk</code> property
+Verification material MUST be expressed in the <code>publicKeyJwk</code> property
```

Comment:
I'm not sure about this. Does this prevent DIDs that do not use `publicKeyJwK` from creating `vc-jose-cose` objects?

Reply:
I'd turn the statement around.  It makes it clear that DIDs used with vc-jose-cose need to support the JWK key representation.  This increases interoperability.

Reply:
I prefer the SHOULD, since I don't plan to be working with JSON in the future, whenever possible.